s3-sink-connector/src/main/java/io/aiven/kafka/connect/s3/config/S3SinkConfig.java

```diff
@@ -565,28 +216,30 @@ public CompressionType getCompressionType() {
         if (Objects.nonNull(getString(FILE_COMPRESSION_TYPE_CONFIG))) {
             return CompressionType.forName(getString(FILE_COMPRESSION_TYPE_CONFIG));
         }
-        if (Objects.nonNull(getString(OUTPUT_COMPRESSION))) {
-            return CompressionType.forName(getString(OUTPUT_COMPRESSION));
+        if (Objects.nonNull(getString(S3ConfigFragment.OUTPUT_COMPRESSION))) {
+            return CompressionType.forName(getString(S3ConfigFragment.OUTPUT_COMPRESSION));
         }
         return CompressionType.GZIP;
     }
 
     /**
-     * Gets the list of output fields. Will check {@link AivenCommonConfig#FORMAT_OUTPUT_FIELDS_CONFIG} and then
-     * {@link #OUTPUT_FIELDS}. If neither is set will create an output field of {@link OutputFieldType#VALUE} and
-     * {@link OutputFieldEncodingType#BASE64}.
+     * Gets the list of output fields. Will check {OutputFormatFragment#FORMAT_OUTPUT_FIELDS_CONFIG} and then
+     * {OutputFormatFragment#OUTPUT_FIELDS}. If neither is set will create an output field of
+     * {@link OutputFieldType#VALUE} and {@link OutputFieldEncodingType#BASE64}.
      *
      * @return The list of output fields. WIll not be {@code null}.
      */
     @Override
     public List<OutputField> getOutputFields() {
```

Comment:
There is an overloaded method for this with an argument  getOutputFields(final String format), which can be deleted I guess